commands/user_e2e_test.go

```diff
@@ -136,3 +136,36 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local can change basic user email", func(c client.Client) {
+		newEmail := "basicuser2@fakedomain.com"
```

Comment:
For consistency, please add the `printer.Clean()` line at the beginning of each test case